src/decide.js

```diff
@@ -0,0 +1,56 @@
+import { autocapture } from './autocapture'
+import { _ } from './utils'
+
+const decide = {
+    instance: null,
+
+    init: function (instance) {
+        this.instance = instance
+        var json_data = JSON.stringify({
+            token: instance.get_config('token'),
+            distinct_id: instance.get_distinct_id(),
+        })
+
+        var encoded_data = _.base64Encode(json_data)
+        instance._send_request(
+            instance.get_config('api_host') + '/decide/',
+            { data: encoded_data },
+            { method: 'POST' },
+            instance._prepare_callback(this.parseDecideResponse)
```

Comment:
Note: `_prepare_callback` is not really needed. posthog-js does not really work unless you use xhr.